templates/users.hamlet

```diff
@@ -1,5 +1,4 @@
-<h1>
-    <center>
+<h1 .text-center>
         Users
```

Comment:
This case needs a different indentation, since you removed a line. :) "Users" should be un-indented. Thanks.